tests/scripts/must_gather.py

```
@@ -157,7 +157,12 @@ def must_gather():
     pod_logs_dir = cluster_dir / "podlogs"
     pod_logs_dir.mkdir(exist_ok=True)
     for pod in cluster_utils.get_running_pods():
-        for container in cluster_utils.get_pod_containers(pod):
+        try:
+            containers = cluster_utils.get_pod_containers(pod)
+        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)